### ISSUE TITLE
[Question]

### ISSUE THREAD

**Issue by Tapas059** [casbin/casbin#811]
**Want to prioritize this issue? Try:**

[![issuehunt-to-marktext](https://github.com/BoostIO/issuehunt-materials/raw/master/v1/issuehunt-button-v1.svg)](https://issuehunt.io/r/casbin/casbin)

------

**What's your scenario? What do you want to achieve?**
 I am trying to check below policy on editor , But in both cases i am getting false. 

**Your model:**
[request_definition]
r = sub, obj, act

[policy_definition]
p = sub, obj, act

[policy_effect]
e = some(where (p.eft == allow))

[matchers]
m = r.sub.Owner == r.obj.Owner && r.sub.Doc in(r.obj.Docs)

```

**Your policy:**

```
{Owner: 'alice', Doc: 1}, { Owner: 'alice', Docs: [1,2] }
{Owner: 'alice', Doc: 1}, { Owner: 'alice', Docs: [3,4]}
```

**Your request(s):**

```
{Owner: 'alice', Doc: 1}, { Owner: 'alice', Docs: [1,2] } ---> false (expected: true)
```

**Comment by hsluoyz** [comment]
@closetool

**Comment by closetool** [comment]
@Tapas059 It should be `false`, for there's an error in your model file, we not support `in` in our system, if you check the error returned by enforcer, you'll receive `Value '[1 2]' cannot be used with the comparator 'in', it is not a number`

**Comment by closetool** [comment]
@Tapas059 
```go
package main

import (
	"fmt"

	"github.com/casbin/casbin/v2"
)

type Sub struct {
	Owner string
	Doc   int
}

type Obj struct {
	Owner string
	Docs  []interface{}
}

func main() {
	e, err := casbin.NewEnforcer("./rbac_model.conf", "./rbac_policy.csv")
	if err != nil {
		panic(err)
	}
	fmt.Println(e.Enforce(Sub{"alice", 1}, Obj{"alice", []interface{}{1, 2}}))
}
```
The array type should be interface.

@hsluoyz We should add more informations about `in` into document.

**Comment by hsluoyz** [comment]
> @hsluoyz We should add more informations about in into document.

@closetool exactly, in fact I already created the issue: https://github.com/casbin/casbin-website/issues/231 , can you add docs for it?

**Comment by closetool** [comment]
@hsluoyz ok

**Comment by Tapas059** [comment]
Thank you @hsluoyz and @closetool  . Let me try out

**Comment by hsluoyz** [comment]
@Tapas059 what language do you use? This repo's issues are for Go.

**Comment by Tapas059** [comment]
Hi @hsluoyz , I am using Node-Casbin . So, i have issue with Node-Casbin . Sorry for the confusion .

**Comment by hsluoyz** [comment]
@Tapas059 we added the docs for "in" (https://casbin.org/docs/en/syntax-for-models#speical-grammer) but that's mostly for Go, I will redirect it.

**Comment by Tapas059** [comment]
Hi @hsluoyz , Any suggestion about node-casbin for above example ?
